charts/resoto/values.yaml

```diff
@@ -37,6 +33,11 @@ securityContext: {}
 #  runAsUser: 1000
 
 resotoworker:
+  image:
+    # -- Image repository
+    repository: somecr.io/someengineering/resotoworker
+    # -- Overrides the image tag whose default is the chart appVersion.
+    tag: ""
```

Comment:
I wonder if we should define the value here as an empty string? Maybe comment it out?

Reply:
Actually it's an override with default pointing to app.Version

Reply:
I know - just wonder if commenting out is more clear.
Not defined: the default kicks in
Defined: the value is used.
Here a value is defined, which is interpreted as undefined...
So TL;DR: I would opt to comment the value out.

Reply:
How about this? We specify a `global.imageTag` which would be default for all tags unless one child tag is explicitly defined @aquamatthias ?